File: swagger/models/get_services_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetServicesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		services
		Contains information about the services.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Services': 'services',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'services': 'Services',
		}

File: swagger/Models/GetServicesResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetServicesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the services.
	 * 
	 * @var Service[] 
	 */
	public $services;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Services' => ['services', 'array', Service::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'services' => ['Services', 'array', Service::class],
			];
	}
}
